elements/rh-alert/rh-alert.css

```diff
@@ -150,20 +126,46 @@ footer ::slotted([slot="actions"]:hover) {
   color: var(--rh-color-interactive-blue-darkest, #003366) !important;
 }
 
-footer ::slotted(rh-button[variant="link"]) {
+footer ::slotted(rh-button[variant="link" i]) {
```

Comment:
typo

Reply:
the `i`? https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#attr_operator_value_i

Reply:
TIL